web/business-registry-dashboard/app/app.config.ts

```diff
@@ -170,6 +135,9 @@ export default defineAppConfig({
         bcGov: 'border-gray-700',
         error: 'border-red-500'
       },
+      size: {
+        lg: 'h-[42px] w-[90px]' // override size from core layer
+      },
```

Comment:
Partly fixes this:

Dev (incorrect)
<img width="216" height="117" alt="image" src="https://github.com/user-attachments/assets/cb6b8c36-f54f-47ea-bfc7-e55a6f15900f" />

Test (correct)
<img width="228" height="102" alt="image" src="https://github.com/user-attachments/assets/cbc0cea4-46fa-45de-9975-8cd2eb2c729d" />